allUnique.js

```diff
@@ -0,0 +1,21 @@
+/*
+Write a program to determine if a string contains only unique characters. Return true if it does and false otherwise.
+
+The string may contain any of the 128 ASCII characters. Characters are case-sensitive, e.g. 'a' and 'A' are considered different characters.
+
+*/
+
+function hasUniqueChars(str){
+    const stringToArray = str.split('');
+    let character;
+    let countTheSameCharacters = 0;
+
+    for(let i = 0; i < stringToArray.length; i++) {
+        character = stringToArray[i];
+
+        if(stringToArray.includes(character, i + 1)) {
+            countTheSameCharacters++;
```

Comment:
nie ma potrzeby inkrementować. Możesz tutaj dać return false;

na dole return true (bo jak nie wy-falsował to znaczy że true) i zmienna character i countTheSame staje się niepotrzebna